### ISSUE TITLE
Possible issue with Carla and latest u-he plugins

### ISSUE THREAD

**Issue by AlbertoZ33n** [falkTX/Carla#799]
Hello Felipe,  I had an issue with Carla (and Ardour 5.12 and Harrison Mixbus) when saving a patch with the newest U-He plugins. The reference of the issue is here:

https://www.reddit.com/r/UheOnLinux/comments/9ecge1/issue_on_preset_saving_host_crashes_in_ardour/

In synthesis saving a patch on any recent u-he plugins crashed Carla, Ardour and Harrison. It did not crash Reaper and Bitwig, that was suspicious. Then, in the same thread someone pointed me out to an Harrison Mixbus patch (which is not in Ardour 5.12 but **it will be in Ardour 6**). This is the link to the ardour patch:

https://tracker.ardour.org/view.php?id=7668

Then I rebuilt Carla with basically the same modification in Carla/source/backend/plugin/CarlaPluginVST2.cpp

```
`        case audioMasterUpdateDisplay:
            // Idle UI if visible
            if (fUI.isVisible)
            // comment the following and no crash anymore ==> 
            //    dispatcher(effEditIdle);

            // Update current program`
```

The above modification in Carla prevents the crash with U-He newest vst's. From what I understood, the "idle" response is not correct in this case, but I'm not really inside the VST stuff to say if this is something good in any case. 
What do you think? 

Thanks, 
Alberto

**Comment by falkTX** [comment]
That makes sense, I have sometimes wondered why other hosts had stuff like that in place.
I do not have a good reason to have the editIdle in there, now there is a good one to NOT have it instead.
I will remove it, thanks!

**Comment by falkTX** [comment]
Fixed in 51f20736b4e0b5d6416f2b7d4051f155f5858fbf
